alvr/client_core/src/decoder.rs

```diff
@@ -7,6 +7,9 @@ use std::time::Duration;
 #[derive(Clone)]
 pub struct DecoderInitConfig {
     pub codec: CodecType,
+    pub width: i32,
+    pub height: i32,
+    pub force_software_decoder: bool,
```

Comment:
This flag should be set from a session setting.

Reply:
Ok, done.